hibernate-core/src/main/java/org/hibernate/internal/util/collections/CollectionHelper.java

```diff
@@ -175,8 +188,7 @@ public static int determineProperSizing(int numberOfElements) {
 	 * @return The created map.
 	 */
 	public static <K, V> ConcurrentHashMap<K, V> concurrentMap(int expectedNumberOfElements, float loadFactor) {
-		final int size = expectedNumberOfElements + 1 + (int) ( expectedNumberOfElements * loadFactor );
-		return new ConcurrentHashMap<>( size, loadFactor );
+		return new ConcurrentHashMap<>( expectedNumberOfElements, loadFactor );
```

Comment:
There is a bug in old implementation. Different than `HashMap` or `HashSet`, `ConcurrentHashMap` will perform sizing computation internally so we don't need to compute on top of that.
![Screen Shot 2020-03-01 at 3 31 38 PM](https://user-images.githubusercontent.com/8211458/75633308-c8993e00-5bd1-11ea-996f-a4592e70419d.png)